dev-go/delve/delve-1.3.2.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit go-module
+EGO_PN="github.com/go-delve/delve"
+SRC_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+DESCRIPTION="A source-level debugger for the Go programming language"
+HOMEPAGE="https://github.com/go-delve/delve"
+
+LICENSE="MIT BSD BSD-2 Apache-2.0"
+SLOT="0"
+
+RESTRICT="test"
+
+src_compile() {
+	go build -ldflags="-X main.Build=${a3e884e}" -o "${S}/dlv" ./cmd/dlv || die
```

Comment:
@oz123 `main.Build` arg should refer to `6c8c450` commit for `1.3.2` version, see https://github.com/go-delve/delve/releases

Reply:
This is now fixed. 10x!